garage/tf/algos/npo.py

```diff
@@ -355,7 +399,8 @@ def _build_policy_loss(self, i):
                 else:
                     raise NotImplementedError('Unknown PGLoss')
```

Comment:
Moved the check to the constructor. I used `in` to check if `pg_loss` is valid because in the constructor we only need to check if an unknown value is passed.